mods/ra/rules/vehicles.yaml

```diff
@@ -714,6 +714,12 @@ QTNK:
 	SelectionDecorations:
 		VisualBounds: 44,38,0,-4
 	MadTank:
+	Detonates:
+		ChargeDelay: 96
+		DetonationDelay: 42
+		ChargeSound: MADCHRG2.AUD
+		DetonationSound: MADEXPLO.AUD
```

Comment:
If you want to change this everywhere, go ahead. It is actually more consistent with the rest of the mod this way. I'm not changing it.